composer.json

```diff
@@ -36,7 +36,10 @@
         "php-http/mock-client": "^0.3.2",
         "symfony/process": "^2.3||^3.0",
         "symfony/http-kernel": "^2.3||^3.0",
-        "phpunit/phpunit": "^5.7 || ^6.0"
+        "phpunit/phpunit": "^5.7 || ^6.0",
+        "toflar/psr6-symfony-http-cache-store": "dev-master",
+        "symfony/lock": "^3.4@beta",
+        "symfony/cache": "^3.4@beta"
```

Comment:
this needs to be ^3.4 and ^1.0 , then we can merge. lets hope symfony 3.4 comes out on time ;-)

Reply:
There is it: https://symfony.com/blog/symfony-3-4-0-released

Reply:
I'll be releasing `toflar/psr6-symfony-http-cache-store` very soon. I even set a reminder in my calendar when November 30th was announced as release date 😄